File: PRAK304-2210817220001-AjengDiahPramesti.py
a = int(input())

if (a > 0) and (a < 10):
    print ('Satuan')
elif (a >= 10) and (a < 20):
    print ('Belasan')
elif (a >= 20) and (a <= 99):
    print ('Puluhan')
elif (a >= 100):
    print ('Anda Menginput Melebihi Limit Bilangan')
elif (a == 0):
    print ('Nol')
else:
    print ('Inputan Salah')

File: PRAK304-2210817220001-AjengDiahPramesti.c
#include <stdio.h>

int main(){
    int a;
    scanf("%d", &a);

    if (a > 0 && a < 10) {
        printf("Satuan", a);
    } else if (a >= 10 && a < 20){
        printf("Belasan", a);
    } else if (a >= 20 && a <= 99){
        printf("Puluhan", a);
    } else if (a >= 100 ){
       printf("Anda Menginput Melebihi Limit Bilangan", a); 
    } else if (a == 0 ){
       printf("Nol", a);    
    }
              
}